modules/appeals_api/spec/config/schemas/v1/10182_json_spec.rb

```diff
@@ -7,10 +7,9 @@
   include SchemaHelpers
   include FixtureHelpers
 
-  let(:json_schema) { read_schema '10182.json' }
-
+  let(:json_schema) { read_schema '10182.json', 'decision_reviews', 'v1' }
   let(:errors) { validator.validate(json).to_a }
-  let(:json) { fixture_as_json 'valid_10182.json', version: 'v1' }
+  let(:json) { fixture_as_json 'decision_reviews/v1/valid_10182.json' }
```

Comment:
If the changes to the fixture helpers here (at the bottom of this changeset) look ok, I'll need to make several follow-up PRs to update the use of this method the same way it's updated here - using a path within the fixtures folder instead of a `version:` and filename